src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -133,6 +156,8 @@ public void end() {
     m_needToRecordBattleStatistics = true;
     m_needToCleanup = true;
     m_needToCheckDefendingPlanesCanLand = true;
+    needToFindRocketTargets = true;
+    needToFireRockets = true;
```

Comment:
A `rocketHelper.end()`  method would be called here if we moved these state variables to the rocket helper.